Packs/AWSCompromisedEC2Response/Playbooks/playbook-AWS_Compromised_EC2_Response.yml

```diff
@@ -0,0 +1,618 @@
+    taskid: 23cceaf7-3c13-4338-8393-063317ce3012
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 25
+        }
+      }
+  "5":
+    id: "5"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "18"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      filters: {}
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      subnetIds: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: Describes one or more of your subnets.
+      id: a3da60ad-27a3-4780-839c-eee9f1ebb97b
+      iscommand: true
+      name: Describe Subnets
+      script: AWS - EC2|||aws-ec2-describe-subnets
+      type: regular
+      version: -1
+    taskid: a3da60ad-27a3-4780-839c-eee9f1ebb97b
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+  "8":
+    id: "8"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "11"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      filters: {}
+      ownerIds: {}
+      region: {}
+      restorableByUserIds: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      snapshotIds:
+        complex:
+          accessor: SnapshotId
+          root: AWS.EC2.Snapshots
+      waiterDelay: {}
+      waiterMaxAttempts: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: A waiter function that waits until the snapshot is complate
+      id: 52ee11b6-09f8-4a51-8cb9-517a970756d1
+      iscommand: true
+      name: Wait for Snapshot to Complete
+      script: AWS - EC2|||aws-ec2-waiter-snapshot_completed
+      type: regular
+      version: -1
+    taskid: 52ee11b6-09f8-4a51-8cb9-517a970756d1
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 720
+        }
+      }
+  "10":
+    id: "10"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "12"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      count:
+        simple: "1"
+      deviceName: {}
+      disableApiTermination: {}
+      ebsDeleteOnTermination: {}
+      ebsEncrypted: {}
+      ebsIops: {}
+      ebsKmsKeyId: {}
+      ebsOptimized: {}
+      ebsSnapshotId: {}
+      ebsVolumeSize: {}
+      ebsVolumeType: {}
+      iamInstanceProfileArn: {}
+      iamInstanceProfileName: {}
+      imageId: {}
+      instanceType: {}
+      keyName: {}
+      launchTemplateId:
+        simple: lt-07795554a551a20cf
+      launchTemplateName: {}
+      launchTemplateVersion: {}
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      securityGroupIds: {}
+      securityGroups: {}
+      subnetId:
+        simple: subnet-034040456c85bd40b
```

Comment:
Please remove the hard-coded `subnetId` argument and replace it with playbook input / some other non hard-coded input